uber/tests/models/test_badge_funcs.py

```diff
@@ -1,16 +1,19 @@
+# TODO: assumes c.STAFF_BADGE and c.SUPPORTER_BADGE are preassigned badge types with configured ranges, should add this to sensible_defaults
```

Comment:
Whoops, good catch, I thought I had removed that when I removed my other TODOs which were taken care of.  I'll push a commit that removes this comment in a minute.